node_modules/oae-authz/lib/api.js

```diff
@@ -490,12 +486,14 @@ var _checkGroupMembershipsForUser = function(userId, groupIds, callback) {
 
         // There are no columns, we need to explode the user's group memberships
         } else {
-            _explodeGroupMemberships(userId, function(err, allMemberships) {
+            _explodeGroupMemberships(userId, function(err, graph) {
                 if (err) {
                     return callback(err);
                 }
 
-                // Return the matching groups
+                // The memberships are the nodes flattened from the graph, with the subject user
+                // removed
+                var allMemberships = _.chain(graph.getNodes()).pluck('id').without(userId).value().sort();
```

Comment:
That seems on par with what I'm trying to do.

With underscore's chaining, I prefer to have each action on a separate line though.